<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileTransferDialogClass</class>
 <widget class="QWidget" name="FileTransferDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTreeView" name="treeViewSource">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="pushButtonTransfer">
     <property name="text">
      <string>Transfer -&gt;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QTreeView" name="treeViewDistination"/>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButtonScanUSB">
     <property name="text">
      <string>Scan USB</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButtonCloseWindowAndEnableUSB">
     <property name="text">
      <string>Close Window And Enable USB</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
